pom.xml

```diff
@@ -219,74 +194,12 @@
   </build>
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>com.google.code.findbugs</groupId>
```

Comment:
Some of these checkers are great (like errorprone). We should keep them, IMO.

Reply:
I can add them at a later stage.